src/app/shared/stack-blitz/stack-blitz-button.ts

```diff
@@ -18,28 +17,31 @@ export class StackBlitzButton {
    * StackBlitz not yet being ready for people with poor network connections or slow devices.
    */
   isDisabled = false;
-  stackBlitzForm: HTMLFormElement | undefined;
   exampleData: ExampleData | undefined;
 
-  @HostListener('mouseover') onMouseOver() {
-    this.isDisabled = !this.stackBlitzForm;
+  /**
+   * Form used to submit the data to Stackblitz.
+   * Important! it needs to be constructed ahead-of-time, because doing so on-demand
+   * will cause Firefox to block the submit as a popup, because it didn't happen within
+   * the same tick as the user interaction.
```

Comment:
```suggestion
   * Important! It needs to be constructed ahead-of-time, because doing so on-demand
   * would cause Firefox to block the submission as a popup. This is because it would't
   * happen within the same tick as the user interaction.
```